Libplanet/Action/TotalSupplyGetter.cs

```diff
@@ -0,0 +1,20 @@
+using Libplanet.Assets;
+
+namespace Libplanet.Action
+{
+    /// <summary>
+    /// An delegate to provide a view of total supply of a currency.
+    /// <para>Gets the total supply of the given <paramref name="currency"/>.
+    /// </para>
+    /// </summary>
+    /// <param name="currency">The currency type to query the total supply.
+    /// </param>
+    /// <returns>The total supply value of the <paramref name="currency"/> in
+    /// <see cref="FungibleAssetValue"/>, or null if there were no former total supply value.
+    /// </returns>
+    /// <remarks>Note that you must not return the value of <paramref name="currency"/> * 0 if
+    /// <see cref="Currency.TotalSupplyTrackable"/> is <value>true</value> and there is no former
```

Comment:
```suggestion
    /// <see cref="Currency.TotalSupplyTrackable"/> is <c>true</c> and there is no former
```
Or `<see langword="true"/>`. We should really decide which should be the standard. 😗

Reply:
Oof. Missed that one.

Reply:
IMO, for single keywords `<see langword="blah"/>` seems more semantical, but there are many uses of `<c/>` in the codebase as well. 🤔

Reply:
`<see langword="foo"/>` is indeed more appropriate, but sadly, DocFX [does not support it properly](https://github.com/dotnet/docfx/issues/7625) at the moment.

Reply:
😢